packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet.factory.ts

```diff
@@ -90,10 +90,12 @@ export class WalletFactory implements IWalletFactory {
 		}
 
 		if (bip === 49 && this.allowsDeriveWithBIP49(wallet)) {
+			/* istanbul ignore next */
 			throw new Exceptions.NotImplemented(this.constructor.name, "fromMnemonic#49");
 		}
 
 		if (bip === 84 && this.allowsDeriveWithBIP84(wallet)) {
+			/* istanbul ignore next */
```

Comment:
Not used yet so I won't bother with coverage. Will need implementing for Bitcoin.